### PR TITLE
issue/62 - Integration with Auto Register

### PR DESCRIPTION
**Pull request by sunnyratilal**
#62

### CODE REVIEW

File: edd-manual-purchases.php [hunk 1]
```diff
@@ -687,6 +687,18 @@ public static function create_payment( $data ) {
 				if ( $user ) {
 					$user_id = $user->ID;
 					$email = $user->user_email;
+				} else {
+					if ( class_exists( 'EDD_Auto_Register' ) ) {
+						$user_info = array(
```
Comment (pippinsplugins) [inline edd-manual-purchases.php]: Let's combine the `else` and `if` into a `elseif`

**Comment by sunnyratilal** [comment]
PR updated - sorry for the delay here

File: edd-manual-purchases.php [hunk 2]
```diff
@@ -687,18 +687,16 @@ public static function create_payment( $data ) {
...
-							'user_info' => array(
-								'id'         => 0,
-								'email'      => $email,
-								'first_name' => $first,
-								'last_name'  => $last
-							)
-						);
-						$user_id = edd_auto_register()->create_user( $user_info );
-					}
+				} elseif ( class_exists( 'EDD_Auto_Register' ) ) {
```
Comment (pippinsplugins) [inline edd-manual-purchases.php]: Formatting needs adjusted here.

**Comment by sunnyratilal** [comment]
Fixed

**Comment by sunnyratilal** [comment]
@pippinsplugins is this good to merge?

**Pull request merged**